egs/wsj/s5/utils/queue.pl

```diff
@@ -553,17 +609,6 @@ ()
 # But we don't know about its exit status.  We'll look at $logfile for this.
 # First work out an array @logfiles of file-locations we need to
```

Comment:
You moved this code re setting `@logfiles` into the if-statement.  This might cause the script to break if the -sync y option is used.